scripts/deploy-gh.sh

```diff
@@ -11,8 +11,6 @@ git merge master --no-edit
 
 # build
 npm run presite
```

Comment:
We don't need it since the examples in the repo have to be updated. There is no way for them to change.